cni/ipam/ipam_test.go

```diff
@@ -203,6 +277,54 @@ var (
 					Expect(err).ShouldNot(HaveOccurred())
 				})
 			})
+
+			Context("When pool is not use", func() {
+				It("Confirm pool was released by succesfully requesting pool", func() {
+					arg.StdinData = getStdinData("0.4.0", "", "")
+					err = plugin.Add(arg)
+					Expect(err).ShouldNot(HaveOccurred())
+				})
+			})
 		})
 	})
 )
+
+func GetFirstAddress() string {
```

Comment:
why this is named as GetFirstAddress

Reply:
Yeah, I guess calling it GetNextAddress would be more appropriate